tournaments/index.js

```diff
@@ -794,6 +794,7 @@ var commands = {
 			if (params.length < 1) {
 				return this.sendReply("Usage: " + cmd + " <minutes|off>");
 			}
+			if (params[0] === 'Infinity' || params[0] === '0') params[0] = 'off';
```

Comment:
Yes, because ``Tournament.prototype.setAutodisqualifyTimeout`` already returns false if ``isNaN(params[0])`` evaluates to true. There's no reason to add ``infinity`` or ``INFINITY`` when they don't affect what the command enters in the modlog.